lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -555,10 +555,16 @@ def get_address(options)
         options[:billing_address] || options[:address]
       end
 
+      def add_token_transaction_type(xml, options)
```

Comment:
💭 Suggestion.

Instead of defining a method to add a single parameter,  you could add it directly.